test/python/test_backend_names.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=invalid-name,missing-docstring,broad-except
+
+"""Backend grouped/deprecated/aliased test."""
+
+from qiskit import register, get_backend, available_backends
+from qiskit.backends.local import QasmSimulatorPy, QasmSimulatorCpp
+from .common import requires_qe_access, QiskitTestCase
+
+
+# Cpp backend required
+try:
+    cpp_backend = QasmSimulatorCpp()
+except FileNotFoundError:
+    _skip_cpp = True
+else:
+    _skip_cpp = False
+
+
+class TestBackendNames(QiskitTestCase):
+    """
+    Test grouped/deprecated/aliased names from providers.
+    """
+
+    def test_local_groups(self):
+        """test local group names are resolved correctly"""
```

Comment:
Please, review the docstrings to make them start with a capital letter.

Reply:
ok.